src/test/resources/application-test.yml

```
@@ -3,4 +3,22 @@ spring:
+        format_sql: true
+        show_sql: true
+        highlight_sql: true
+
+app:
+  kakao:
+    auth:
+      client: asdf1234asdf1234asdf1234asdf1234asdf1234
+      redirect: http://localhost:8080/auth/login/kakao
+  jwt:
+    access-expire-ms: 360000
+    refresh-expire-ms : 3600000
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)